addons/mail/static/tests/web/attachment_box_tests.js

```diff
@@ -245,3 +245,56 @@ QUnit.test("attachment box auto-closed on switch to record wih no attachments",
     await click(".o_pager_next");
     await contains(".o-mail-AttachmentBox", { count: 0 });
 });
+
+QUnit.test(
+    "open attachment box should remain open after adding a new attachment",
+    async (assert) => {
+        const pyEnv = await startServer();
+        const recordId = pyEnv["mail.test.simple.main.attachment"].create({});
```

Comment:
This model is only available in `test_mail` module, either move the test or just use an available thread model (like partner form view).